Flick/Views/MediaListHeaderView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  MediaListHeaderView.swift
+//  Flick
+//
+//  Created by HAIYING WENG on 5/31/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import UIKit
+
+class MediaListHeaderView: UICollectionReusableView {
+    
+    // MARK: - Private View Vars
+    private let addButton = UIButton()
+    private let containerView = UIView()
+    private let editButton = UIButton()
+    private let roundView = RoundTopView(hasShadow: true)
+    private let sortButton = UIButton()
+
+    // MARK: - Private Data Vars
+    private let buttonSize = CGSize(width: 44, height: 44)
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .offWhite
+
+        containerView.addSubview(roundView)
+        containerView.clipsToBounds = true
+        addSubview(containerView)
```

Comment:
Why do we need this container view?

Reply:
This might sound kinda wack. I used `containerView` to clip `roundTopView` to bounds of `containerView`. I had to do this because `roundTopView` needed height > ~90 because otherwise it shrinks in width. But height > 90 would make the header look too large so I made `containerView` to kinda show only part of the `roundTopView`

Reply:
lol thats okay.